quiche/src/lib.rs

```diff
@@ -569,18 +569,82 @@ pub enum Error {
     KeyUpdate,
 }
 
+/// Error codes as defined in https://www.rfc-editor.org/rfc/rfc9000.html#name-error-codes.
+pub enum QuicTransportErrorWireCode {
+    /// An endpoint uses this with CONNECTION_CLOSE to signal that the connection is being closed
+    /// abruptly in the absence of any error.
+    NoError = 0x00,
```

Comment:
I know the RFC has leading zeroes but that's for presentation purposes, may as well remove them from code.